snap/quota/resources.go

```diff
@@ -63,6 +81,7 @@ type Resources struct {
 	CPU     *ResourceCPU     `json:"cpu,omitempty"`
 	CPUSet  *ResourceCPUSet  `json:"cpu-set,omitempty"`
 	Threads *ResourceThreads `json:"thread,omitempty"`
+	Journal *ResourceJournal `json:"journal,omitempty"`
```

Comment:
`omitempty`in this case will only hide the structure if the pointer is `nil`, otherwise it will output an empty structure. You can see an example here: https://play.golang.com/p/XoVnGvrr_Bv 

The namespace reffered to here is the log namespace for journald